test/propagatortest.jl

```diff
@@ -0,0 +1,33 @@
+using Base.Test
```

Comment:
But only if you run this file individually. `runtests.jl` already loads `Base.Test`. If you want this file to run alone, then you have to load `QuBase` and `QuDynamics` as well.